lambdas/data-migration2/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@cumulus/data-migration2",
+  "version": "3.0.0",
+  "description": "A Lambda function used for doing data migrations of executions",
```

Comment:
Nit: ultimately it won't just be for migrating executions, but also granules/files.